docs/tutorial/troubleshooting.md

```diff
@@ -18,7 +18,7 @@ If you are working in a directory containing the repo folder, `pynxtools`, renam
 
 ```
 .
-├── pynxtools2                    # Renamed repo folder
+├── pynxtools2             # Renamed repo folder
```

Comment:
@sherjeelshabih I think this structure is not up to date anymore. Do you know if this guide still works?

Reply:
In order to no longer block it i) surplus given the fact that the paragraph above has no contextualization in which execution/CLI calling context such error was observed I would remove the paragraph, if this is really so important we can also just replace the paragraph now with
<!---commit ID where this troubleshooting was still available to help us later find it efficiently-->

Reply:
This troubleshooting guide is not even working anymore, as you can absolutely do `from pynxtools import get_nexus_version`. So I simply removed it.